pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -238,8 +238,7 @@ stdenv.mkDerivation rec {
       a corporation.
     '';
     license = licenses.gpl3;
-    platforms = platforms.all;
```

Comment:
Why drop `meta.platforms`?  The result is removal of some jobs on Hydra, e.g. https://hydra.nixos.org/eval/1771320#tabs-removed

Reply:
Also both `*-darwin` platforms get removed, but that's not visible in this jobset (it's a NixOS one).

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/release-lib.nix#L150